### PR TITLE
Various updates for PHP 7

### PR DESCRIPTION
**Pull request by WillJW**

### CODE REVIEW

**Comment by maxakropolis** [comment]
Build was failing so have changed php version on semaphore to 7.0

**Comment by ziadoz** [comment]
Everything seems okay to me. 👍

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6"
-  },
-  "require": {
+    "php": ">=7.0",
     "phpunit/phpunit": "5.*"
```
Comment (gabriel403) [inline composer.json]: I guess this should be require-dev, not something to worry about here though

File: src/Base.php [hunk 2]
```diff
@@ -21,11 +19,11 @@ public function setName($name)
      * Populates flag elements - only `$online` is required. available is
      *
      * @param $online
-     * @param null $searchable
-     * @param null $available This is deprecated according to the XSD, but used so we support it
-     * @param null $both
+     * @param $searchable
```
Comment (gabriel403) [inline src/Base.php]: We can probably dump the whole phpdoc

**Comment by gabriel403** [comment]
Looks good :+1:

**Comment by weshooper** [comment]
Nice 👍 Lets merge and flag a new (major) release...

File: composer.json (hunk 1 above)
Comment (ziadoz) [inline composer.json]: @gabriel403 @WillJW Actually, I think we should sort this here. Might as well do it now.

Comment (WillJW) [inline composer.json]: @gabriel403 @ziadoz Should this be `require-dev` **instead of** `require`, for both php and phpunit?

Comment (ziadoz) [inline composer.json]: @WillJW The PHPUnit bit belongs in `require-dev`. The PHP part should be fine where it is.

Comment (WillJW) [inline composer.json]: @ziadoz I've done that and now Semaphore's failed. Any ideas?

Comment (ziadoz) [inline composer.json]: @WillJW I guess the build instructions might need updating. I'll take a look.

**Comment by ziadoz** [comment]
Looks good. 👍

**Pull request merged**